src/event.rs

```diff
@@ -301,9 +301,11 @@ impl<N: Notify> Processor<N> {
                         processor.received_char(c);
                     },
                     MouseInput { state, button, modifiers, .. } => {
-                        *hide_cursor = false;
-                        processor.mouse_input(state, button, modifiers);
-                        processor.ctx.terminal.dirty = true;
+                        if !cfg!(target_os = "macos") || *window_is_focused {
```

Comment:
Hey I think there has been a misunderstanding here. This should be available on all platforms, not only MacOS.

I think MacOS doesn't do anything significantly different than floating WMs on linux/windows. So consistency across platforms is probably a good idea.

Reply:
Done.

Reply:
Is there anything else preventing us from merging this?